amity/amity.py

```diff
@@ -0,0 +1,255 @@
+import os, sys, inspect
+from sqlalchemy.orm import relationship, backref
+currentdir = os.path.dirname(os.path.abspath(inspect. \
+    getfile(inspect.currentframe())))
+parentdir = os.path.dirname(currentdir)
+sys.path.insert(0, parentdir) 
+from rooms import Room, LivingSpace, Office 
+from persons import Person, Fellow, Staff
+import random
+
+class Amity(object):
+    """Defines the Amity class"""
+    
+    def __init__(self, office_capacity=6, living_space_capacity=4,
+        no_of_occupants = 0):
+        self.office_capacity = office_capacity
+        self.living_space_capacity = living_space_capacity
+        self.no_of_occupants = no_of_occupants
+        
+    def create_room(self, identifier, name, floor, room_type):
+        """Create room object and return"""
+        if room_type == "office":
+            new_room = Office(
+                                identifier=identifier, 
+                                name=name, 
+                                floor=floor,
+                                no_of_occupants = self.no_of_occupants,
```

Comment:
**Code Quality - Nitpick:** There should be no space before and after `=`; only when calling a function/method.